includes/handler.inc

```diff
@@ -182,7 +182,7 @@ function islandora_oai_object_response_xml($params) {
         $link_element = $dom->createElementNS($record->record_namespace, "{$record->record_prefix}:{$field}", $object_url);
       }
       else {
-        $link_element = $dom->createElement("$field", $object_url);
+        $link_element = $dom->createElementNS('http://purl.org/dc/elements/1.1/', "$field", $object_url);
```

Comment:
Hey @bradspry,

I feel like this changes what the intent of the checkbox is trying to do in the fact that when it's not set it's relying on the default namespace of the record XML being created for the identifier field. I don't seem to get/see this problem when I test. Would it it be possible to update the JIRA ticket with a full list of steps and or sample metadata to test against?

Thanks,

Jordan